### PR TITLE
LPS-199390, LPS-199391, LPS-199392, LPS-199393 | Create tests for Kaleo TableReferenceDefinition tests

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPS-199390
https://liferay.atlassian.net/browse/LPS-199391
https://liferay.atlassian.net/browse/LPS-199392
https://liferay.atlassian.net/browse/LPS-199393

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/change/tracking/test/KaleoInstanceTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+				WorkflowConstants.CONTEXT_ENTRY_CLASS_PK,
+				String.valueOf(_blogsEntry.getEntryId())
+			).put(
+				WorkflowConstants.CONTEXT_SERVICE_CONTEXT,
+				(Serializable)_serviceContext
+			).build(),
+			_serviceContext);
+	}
+
+	private static BlogsEntry _blogsEntry;
```
Comment (pei-jung) [inline modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/change/tracking/test/KaleoInstanceTableReferenceDefinitionTest.java:78]: we don't need to make the blogsEntry static.

**Review by pei-jung — CHANGES_REQUESTED** [review]
@brookedalton there's a lot of similar setups for the tests. Let's extract the logic to add the models in a base class to reduce duplicate codes. See `BaseKaleoLocalServiceTestCase` as an example. Please include the tests from https://github.com/liferay-publications/liferay-portal/pull/774. Thank you!

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
Very nicely done, @brookedalton! 👍 

Please see my minor changes in https://github.com/liferay-publications/liferay-portal/pull/858.

**Pull request closed without merging**